src/router/routes/draft/finish.ts

```diff
@@ -81,7 +84,8 @@ const handler = async (req: Request, res: Response) => {
       });
     }
 
-    if (typeof draft.owner !== 'string' && draft.owner?.id !== req.user.id) {
+    const draftOwnerId = typeof draft.owner !== 'string' ? draft.owner?.id : draft.owner;
```

Comment:
This change ended up being unnecessary, I thought I was going to use draftOwnerId in sendDraftNotification but turns out for addNotification I needed the objects